src/Auth0.OidcClient.WPF/PlatformWebView.cs

```diff
@@ -43,64 +43,52 @@ public PlatformWebView(string title = "Authenticating ...", int width = 1024, in
                 Height = height
             })
         {
-            _shouldCloseWindow = true;
         }
 
         public async Task<BrowserResult> InvokeAsync(BrowserOptions options)
         {
             var window = _windowFactory.Invoke();
+            var result = new BrowserResult
+            {
+                ResultType = BrowserResultType.UserCancel
+            };
 
             try
             {
-                var grid = new Grid();
-
-                window.Content = grid;
                 var browser = new WebBrowser();
-
                 var signal = new SemaphoreSlim(0, 1);
 
-                var result = new BrowserResult
-                {
-                    ResultType = BrowserResultType.UserCancel
-                };
-
                 window.Closed += (o, e) =>
                 {
                     signal.Release();
                 };
-                browser.LoadCompleted += (sender, args) =>
+
+                browser.LoadCompleted += (s, e) =>
```

Comment:
why change the argument name? it was very clear what they were before

Reply:
Event handler args are always named 'e' on .NET. It's a standard practice that even Microsoft promotes e.g. https://docs.microsoft.com/en-us/dotnet/api/system.eventhandler?view=netframework-4.8

I could revert sender back if you prefer - that practice is less well-spread.

Reply:
nope, e is fine!